kin-sdk-core/src/androidTest/java/kin/sdk/core/KinAccountTest.java

```diff
@@ -114,58 +108,48 @@ public void getPublicKey_deletedAccount()
         assertEquals("", publicAddress);
     }
 
-    @Test
-    public void sendTransactionSync_insufficientBalance() {
-        try {
-            kinAccount.sendTransactionSync(TO_ADDRESS, PASSPHRASE, new BigDecimal(1));
-        } catch (Exception e) {
-            assertTrue(e instanceof InsufficientBalanceException);
-        }
+    @Test(expected = InsufficientBalanceException.class)
+    public void sendTransactionSync_insufficientBalance() throws Exception {
+        kinAccount.sendTransactionSync(TO_ADDRESS, PASSPHRASE, new BigDecimal(1));
     }
 
-    @Test
-    public void sendTransactionSync_negativeAmount() {
-        try {
-            kinAccount.sendTransactionSync(TO_ADDRESS, PASSPHRASE, new BigDecimal(-1));
-        } catch (Exception e) {
-            assertTrue(e instanceof OperationFailedException);
-            assertEquals("Amount can't be negative", e.getMessage());
-        }
+    @Test(expected = OperationFailedException.class)
+    public void sendTransactionSync_negativeAmount() throws Exception {
+        kinAccount.sendTransactionSync(TO_ADDRESS, PASSPHRASE, new BigDecimal(-1));
```

Comment:
we want to make sure also the message is correct.

Reply:
fixed, now considering the message also